cmake/modules/OmrDetectSystemInformation.cmake

```diff
@@ -83,11 +83,13 @@ macro(omr_detect_system_information)
 
    if(OMR_HOST_OS STREQUAL "zos")
       # ZOS specifics
+      set(OMR_TOOLCONFIG "gnu")
    endif()
 
 
    if(OMR_HOST_OS STREQUAL "osx")
       # OSX specifics
+      set(OMR_TOOLCONFIG "gnu")
```

Comment:
I believe the long-run intention is that gnu will cover all gnu-ish toolchains, including clang. This is just the smallest change I could make to get osx/cmake building.